configure

```diff
@@ -144,10 +144,13 @@ def main():
     parser.add_argument('--linker', dest='linker', type=str, default=None, help="which linker program to use")
     parser.add_argument('--pkg-config', dest='pkg_config', type=str, default="pkg-config")
     parser.add_argument('--bootstrap', dest='bootstrap', type=str, default=os.path.join(os.path.dirname(os.path.abspath(__file__)), "bootstrap"), help="a directory for bootstrapping commonly used headers and libraries not associated directly with the target operating system")
+    parser.add_argument('--libdir-suffix', dest='libdir_suffix', type=str, default=None, help="library folder suffix")
     parser.add_argument('--reconfigure', dest='reconfigure', action="store_true", help="re-generate the build script given the previous configuration")
     parser.add_argument('-v', '--verbose', dest='verbose', action="store_true")
     args, extras = parser.parse_known_args()
     
+    config.libdir = "lib" + args.libdir_suffix
+    
```

Comment:
If the default is None, concatenation without that flag passed will throw an exception. Perhaps check?

Reply:
True, it will throw an exception. Changed for a explicit default before the parse, and then check before concatenating the string.